css/reviews.css

```diff
@@ -0,0 +1,435 @@
+    color: white;
+}
+
+h1 {
+    font-size: 48px;
+}
+
+h2 {
+    font-family: "lato", sans-serif;
+    font-size: 39px;
+}
+
+h3 {
+    font-size: 31px;
+}
+
+h4 {
+    font-size: 25px;
+}
+
+h5 {
+    font-size: 20px;
+}
+
+h6 {
+    font-size: 16px;
+}
+
+p {
+    font-family: "lato", sans-serif;
+    font-size: 20px;
+    color: white;
+}
+
+label {
+    font-family: "lato", sans-serif;
+}
+
+.p2 {
+    font-size: 16px;
+}
+
+.p-mobile {
+    font-size: 14px;
+}
+
+.polaroid-font {
+    font-family: "caveat", sans-serif;
+    color: black;
+}
+
+/*header*/
+
+nav a {
+    color: white;
+    font-size: 16px;
+    padding: 0 10px;
+    position: relative;
+    transition: 0.7s ease;
+}
+
+nav a:after{
+    content: "";
+    background-color: #FFB24B;
+    height: 2px;
+    width: 0;
+    left: 0;
+    bottom: -10px;
+    transition: 0.3s;
+    position: absolute;
+}
+nav a:hover {
+    color: #FFB24B;
+}
+
+nav a:hover:after {
+    width: 100%;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+}
+
+.logo {
+    height: 34px;
+    width: 70px;
+    margin: 40px 40px;
+}
+
+nav {
+    display: flex;
+    width: 100%;
+    height: 114px;
+    justify-content: center;
+    align-items: center;
+    gap: 40px;
+}
+
+.link-home {
+    cursor: pointer;
+}
+
+.hamburger {
+    cursor: pointer;
+    display: none;
+}
+
+.bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: white;
+}
+
+/* body */
+.carousel {
+    position: relative;
+    overflow: hidden;
+    height: 60vh;
+}
+
+.center-this {
+    display: flex;
+    justify-content: center;
+}
+
+.slider {
+    display: flex;
+    transition: transform 0.5s ease;
+}
+
+/* Review-cards */
+.review-card {
+    display: flex;
+    align-items: center;
+    border: 1px solid #FFB24B;
+    border-radius: 16px;
+    margin: 20px;
+    width: 365px;
+    height: 214px;
+}
+
+.person-info {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: white;
+    border-top-left-radius: 16px;
+    padding-bottom: 8px;
+    border-bottom-left-radius: 16px;
+}
+
+.person-image {
+    margin-bottom: 10px;
+}
+
+.person-image img {
+    width: 100px;
+    height: 100px;
+}
+
+.review-content {
+    background-color: black;
+    border-top-right-radius: 16px;
+    border-bottom-right-radius: 16px;
+    padding: 5px;  
+}
+
+.review-text {
+    font-size: 16px;
+    margin-bottom: 10px;
+    color: white;
+    margin-left: 16px;
+}
+
+.rating {
+    font-size: 18px;
+}
+
+.star {
+    color: gold;
+}
+
+.slider img {
+    width: calc(33.33% - 10px);
+    margin-right: 16px;
+    width: 100px;
+    height: 150px;
+    margin-left: 16px;
+    margin-top: 16px;
+}
+
+.controls {
+    position: absolute;
+    bottom: 10px;
+    left: 50%;
+    transform: translateX(-50%);
+    display: flex;
+}
+
+button {
```

Comment:
you can just give buttons .btn class and it will pull code from styles.css as long as you link it